src/main/java/com/dao/UserDaoImpl.java

```diff
@@ -40,11 +39,20 @@ public void add(User user) {
         }
     }
 
+    @Override
+    public User get(Long id) {
```

Comment:
Maybe it is better to return Optional<User> from DB

Reply:
> Maybe it is better to return Optional from DB

in this case, it's not obligatory